### ISSUE TITLE
No cookie svg?

### ISSUE THREAD

**Issue by scottynorris** [manucaralmo/GlowCookies#62]
Not sure if it's default behaviour, but the little cookie icon in the "accept" button isn't appearing. I like that little touch! How can I get that working?

Other than this, perfect pop-up and very user friendly - thank you!

**Comment by chrisonline** [comment]
Yes it is missing. Have made the change and created a pull request:
https://github.com/manucaralmo/GlowCookies/pull/65
